synapse/rest/client/v2_alpha/account.py

```diff
@@ -439,7 +441,8 @@ def on_POST(self, request):
             "email", body["email"]
         )
 
-        if existing_user_id is not None:
+        # Allow this MSISDN to be rebound if the requester owns it
+        if existing_user_id != requester.user.to_string():
```

Comment:
Can we add this check to the bind servlet too for sanity please?

Reply:
What about `/account/3pid/*/requestToken`?

Reply:
I don't understand the question I'm afraid

Reply:
Should we add the same check to https://matrix.org/docs/spec/client_server/unstable#post-matrix-client-r0-account-3pid-email-requesttoken and the MSISDN equivalent?

Reply:
I guess? Is that what the MSC says?

Reply:
yes

Reply:
So adding the check to the /bind servlet is what's breaking Sytest, in that it didn't used to check for already bound emails but now it does. I suppose this isn't really part of the MSC so should we do it here?